src/odata-to-abstract-sql.ts

```diff
@@ -1110,7 +1103,7 @@ export class OData2AbstractSQL {
 			throw new SyntaxError('Failed to match a Date entry');
 		}
 	}
-	DurationMatch(match: DurationNode): AbstractSqlType | undefined {
+	DurationMatch(match: DurationNode[1]): AbstractSqlType | undefined {
```

Comment:
This was brought up by the lodash typings bump, DurationNode is `['Duration', {...}]` and we specifically expect the object part of a DurationNode here